fsw/platform_inc/generic_torquer_app_platform_cfg.h

```diff
@@ -32,6 +32,14 @@
     #define GENERIC_TORQUER_CFG_HANDLE           29 
     #define GENERIC_TORQUER_CFG_BAUDRATE_HZ      115200
     #define GENERIC_TORQUER_CFG_MS_TIMEOUT       250
+
+    #define TRQ_PERIOD               50000000
+    #define TRQ_MIN_PERCENT          0
+    #define TRQ_MAX_PERCENT          100
+    #define TRQ1_GPIO                15
+    #define TRQ2_GPIO                16
+    #define TRQ3_GPIO                17
```

Comment:
Should remove all GPIO use for this initial release - keep enable / disable in the app, but only maintain internal state and not drive GPIO pins.